Source/WebKit/Scripts/copy-webcontent-resources-to-private-headers.sh

```diff
@@ -39,7 +39,9 @@ PROCESSED_INFOPLIST="${BUILT_PRODUCTS_DIR}/${INFOPLIST_PATH}"
 UNPROCESSED_INFOPLIST="${INFOPLIST_FILE}"
 COPIED_INFOPLIST="${WEB_CONTENT_RESOURCES_PATH}/Info-WebContent.plist"
 echo "Copying Info.plist from ${UNPROCESSED_INFOPLIST} to ${COPIED_INFOPLIST}"
```

Comment:
You should move this line inside the if block too.

Reply:
Ah, good point, will fix.

Thanks for reviewing!